### DISCUSSION TITLE
Default keybindings for inner and outer-of-thing

### DISCUSSION THREAD

**Discussion by joshcho** [meow-edit/meow#183] [Ideas]
I found it strange that there were another set of keybindings that I had to keep in mind for `meow-inner-of-thing` and `meow-outer-of-thing`. I think more sensible default keybindings would be the symbol corresponding to the thing: ( or ) for `'round`, [ or ] for `'square`, etc.

**Comment by DogLooksGood** [comment]
There's no best keybinding in my opinion, so it is up to you.  You can consider the value of `meow-char-thing-table` an exapmle, just set it  according to your taste.  Updating the default value will break others' configuration, so...

```
(setq meow-char-thing-table
  '((?( . round)
    (?) . round)
    ...))
```
